packages/bash.rb

```diff
@@ -51,9 +41,9 @@ def self.install
     FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/etc/bash.d/"
     @bashenv = <<~BASHEOF
       # Make Chromebrew's version of bash start automatically
-      # This currently seems to be broken, uncomment at your own risk
-      # #{CREW_PREFIX}/bin/bash
-      # PS1=$PS1
+      if [[ "$(coreutils --coreutils-prog=readlink "/proc/$$/exe")" != '#{CREW_PREFIX}/bin/bash' ]]; then
```

Comment:
`'#{CREW_PREFIX}/bin/bash'` should be changed to `"#{CREW_PREFIX}/bin/bash"`, otherwise `#{CREW_PREFIX}` will not be replaced.

Reply:
Actually this doesn't matter, because it is in a ruby heredoc. - e.g.
```
# Make Chromebrew's version of bash start automatically
if [[ "$(coreutils --coreutils-prog=readlink "/proc/$$/exe")" != '/usr/local/bin/bash' ]]; then
  exec /usr/local/bin/bash
fi
```